app/src/debug/java/net/squanchy/support/debug/DebugActivity.kt

```diff
@@ -72,18 +73,18 @@ class DebugActivity : AppCompatActivity() {
             endTime = end,
             title = "A very interesting talk",
             place = createPlace(),
-            experienceLevel = Optional.of(ExperienceLevel.ADVANCED),
+            experienceLevel = Option(ExperienceLevel.ADVANCED),
             speakers = createTalkSpeakers(),
             type = Event.Type.TALK,
             favorited = true,
-            description = Optional.absent(),
-            track = Optional.of(createTrack()),
+            description = None,
+            track = Option(createTrack()),
             timeZone = DateTimeZone.forID("Europe/Rome")
         )
     }
 
-    private fun createPlace(): Optional<Place> = Optional.of(
-        Place("1", "That room over there", Optional.absent())
+    private fun createPlace(): Option<Place> = Option(
+        Place("1", "That room over there", Option.empty())
```

Comment:
What's the difference between `None` and `Option.empty()`?

Reply:
Using `None` directly is a bit more white-box I suppose. Should replace usages of `None` with `empty()`

Reply:
e479f844abd1c6c981dfbef3a45439434f49648d replaced